web-access-control/protected-operation/read-access-bob.feature

```diff
@@ -1,119 +1,210 @@
     """
-    # Create 3 test resources with read access for Bob
-    * def testsR = callonce createResources ['read']
-    # Create 3 test resources with append, write, control access for Bob
-    * def testsAWC = callonce createResources ['append', 'write', 'control']
 
-  Scenario Outline: <agent> read a <type> resource (<method>) to which Bob has <mode> access
-    Given url tests<mode>[type].url
-    And headers authHeaders(method, tests<mode>[type].url, public)
+  Scenario Outline: <agent> <result> read a <type> resource (<method>), when Bob has <container> access to the container and <resource> access to the resource
+    * def testResource = createResource(container, resource, type, agent)
+    Given url testResource.url
+    And headers authHeaders(method, testResource.url, agent)
     When method <method>
     Then status <status>
     Examples:
-      | agent         | type      | mode | method | public! | status |
-      | Bob can       | plain     | R    | GET    | false   | 200    |
-      | Bob can       | rdf       | R    | GET    | false   | 200    |
-      | Bob can       | container | R    | GET    | false   | 200    |
-      | Bob can       | plain     | R    | HEAD   | false   | 200    |
-      | Bob can       | rdf       | R    | HEAD   | false   | 200    |
-      | Bob can       | container | R    | HEAD   | false   | 200    |
-      | Public cannot | plain     | R    | GET    | true    | 401    |
-      | Public cannot | rdf       | R    | GET    | true    | 401    |
-      | Public cannot | container | R    | GET    | true    | 401    |
-      | Public cannot | plain     | R    | HEAD   | true    | 401    |
-      | Public cannot | rdf       | R    | HEAD   | true    | 401    |
-      | Public cannot | container | R    | HEAD   | true    | 401    |
-      | Bob cannot    | plain     | AWC  | GET    | false   | 403    |
-      | Bob cannot    | rdf       | AWC  | GET    | false   | 403    |
-      | Bob cannot    | container | AWC  | GET    | false   | 403    |
-      | Bob cannot    | plain     | AWC  | HEAD   | false   | 403    |
-      | Bob cannot    | rdf       | AWC  | HEAD   | false   | 403    |
-      | Bob cannot    | container | AWC  | HEAD   | false   | 403    |
-      | Public cannot | plain     | AWC  | GET    | true    | 401    |
-      | Public cannot | rdf       | AWC  | GET    | true    | 401    |
-      | Public cannot | container | AWC  | GET    | true    | 401    |
-      | Public cannot | plain     | AWC  | HEAD   | true    | 401    |
-      | Public cannot | rdf       | AWC  | HEAD   | true    | 401    |
-      | Public cannot | container | AWC  | HEAD   | true    | 401    |
+      | agent  | result | method  | type      | container | resource  | status |
+      | Bob    | can    | GET     | plain     | no        | R         | 200    |
+      | Bob    | can    | GET     | plain     | R         | inherited | 200    |
+      | Bob    | can    | GET     | fictive   | R         | inherited | 404    |
```

Comment:
```suggestion
      | Bob    | cannot | GET     | fictive   | R         | inherited | 404    |
```
There are other examples of this but I think the scenario description needs to end up describing what happened. If we expect a 404 then in this scenario Bob cannot access the resource

Reply:
Ohhhh yes, that is true. Thank you. I fixed some of those, but I forgot to check all of them it seems. I will go through them again.